classes/pcdm.py

```diff
@@ -232,7 +233,10 @@ def components(self):
         return [ obj for (rel, obj) in self.linked_objects if rel == pcdm.hasMember ]
 
     def ordered_components(self):
-        return [ obj for obj in self.components() if obj.ordered ]
+        orig_list = [ obj for obj in self.components() if obj.ordered ]
+        sort_key = self.sequence_attr[1]
+        sorted_list = sorted(orig_list, key=attrgetter(k))
```

Comment:
Should the `k` argument to `attrgetter` be `sort_key` instead?

Reply:
Whoops, yes -- I was trying to revise to make the variable names more obvious. I'll update the PR.